test/support/commands.js

```diff
@@ -1,2 +1 @@
-Cypress.Commands.add('getByTestId', testId => cy.get(`[data-testid="${testId}"]`));
-Cypress.Commands.add('getByTitle', title => cy.get(`[title="${title}"]`));
+import 'cypress-testing-library/add-commands';
```

Comment:
We decided put adoption of this library on hold for now. We would prefer to use `cy` methods directly or via custom commands.